src/components/LeftPanel.tsx

```
@@ -235,19 +174,102 @@ function ItineraryPanel({
+          const segmentsForThisDay = routeSegmentsByDay[layer.id] || [];
+          const containerBodyClasses = `transition-opacity duration-300 ${isDayVisible ? 'opacity-100' : 'opacity-40 pointer-events-none'}`;
+
+          // 각 날짜별 접힘/펼침 상태를 관리하기 위해 ItineraryPanel 내부에 상태를 만듭니다.
+          // 더 복잡한 상태 관리가 필요하면 이 로직을 부모 컴포넌트로 올릴 수 있습니다.
+          const [isCollapsed, setIsCollapsed] = useState(false);
+          const { setNodeRef } = useDroppable({ id: layer.id });
+
+          return (
+            <div key={layer.id} className="border-b pb-2">
+              {/* Header: 토글 스위치, 날짜, 버튼들 */}
+              <div ref={setNodeRef} className="flex items-center mb-2 gap-2">
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)